src/Honeybee_addHBGlz.py

```diff
@@ -147,6 +147,9 @@ def addPotentialChildSurface(HBSurface):
                         return
     
                 # add it to the base surface
+                HBSurface.BC= "Outdoors"
+                HBSurface.sunExposure = "SunExposed"
+                HBSurface.windExposure = "WindExposed"
```

Comment:
This is not exactly how I imagined the component would work if someone adds a glazing surface in between two adjacent Surfaces (each with a Surface boundary condition). For this case, I feel like we don't want this component to set it to Outdoors but rather treat the added glazing as interior. Maybe this is too difficult, though, since it's hard to know when someone adds it to one interior Surface but not the adjacent one.

What if we instead did a check first to see if the BC of the parent surface is `Outdoors` (before running the child-surface check) instead of changing it to `Outdoors` after a child-surface match is found?

Reply:
Agreed, I totally missed the interior condition.
The use case I was trying to solve is:
In a smaller scale box model, all exterior walls are set to Adiabatic, and EnergyPlus will fail to run with windows on adiabatic walls. It has to be Outdoors or interior Surface.

Yes, this might be hard to do for both conditions. Maybe just throw an error when users trying to add windows on adiabatic walls, is an easiest way.

Reply:
Thanks for confirming and I agree that this needs some type of a solution. I think we'll just ignore the Surface boundary condition case and, if someone wants an interior window, they will just have to add it before solving adjacency.

So I am in favor of either throwing an error or just ignoring the adding of all glazing to any surfaces that are not Outdoors. I trust your judgement for which one you feel is best to implement but I feel that your currently-implemented solution is too implicit rather than explicit. I can easily see some users having the boundary condition changed on them without them realizing what happened.